### ISSUE TITLE
Fix perfdata output for memory checks (and others with percents)

### ISSUE THREAD

**Issue by jomann09** [NagiosEnterprises/ncpa#713]
When perfdata output is given the warn/critical values are rounded, so it looks like this:

```
WARNING: Used memory was 75.30 % (Available: 7.89 GiB, Total: 31.89 GiB, Free: 7.89 GiB, Used: 24.00 GiB) | 'available'=7.86GiB;22;24; 'total'=31.89GiB;22;24; 'free'=7.86GiB;22;24; 'used'=24.03GiB;22;24;
```

We used warn of 70% and crit of 76% so it was technically still a warning value but because the perfdata does not have decimals for the warn/crit values, if you were looking at the warn/crit values in the perfdata it looks like it's in critical.

**Comment by jomann09** [comment]
There is actually a bigger problem than this it seems, the warning/critical values really shouldn't be applied to the perfdata, because in this specific check the warn/crit is only applied to the %, which is actually a combination of a few values.

**Comment by jomann09** [comment]
Okay so I am assuming some people are going to wonder what happened here, since there is now going to be no warning/critical values given on these larger checks (such as interface/<name> and disk/mount/<name>) checks as well as with the memory/virtual check. I noticed that the _warning and configuration values are being applied to a specific piece of data_, but we were treating the warning and critical values as if they should be applied to all data points for some reason. That does not give an accurate representation of the check result, since the only result that matters is that the check is OK or not.

Example:
The check I gave in the original post was using `-w 22 -c 24` and the warning and critical values are the percentage of memory used, the first value below, which is 75.30%.
```
WARNING: Used memory was 75.30 % (Available: 7.89 GiB, Total: 31.89 GiB, Free: 7.89 GiB, Used: 24.00 GiB) | 'available'=7.86GiB;22;24; 'total'=31.89GiB;22;24; 'free'=7.86GiB;22;24; 'used'=24.03GiB;22;24;
```
The perfdata warning/critical values are showing `22;24;` and those are not accurate values because 1) the warning/critical values are going to only apply to the percent in the output, meaning that even if 24% of the free value is passed, the check will not go into critical. Also, if 22GiB of free value is passed, it will also not go into critical. So these values are actually inaccurate and should be removed.

In NCPA 2.3.1 these values **will be removed**. Only the data that the warning/critical values correspond to will now show the warning/critical values.
